example-application/api-under-test.js

```diff
@@ -4,7 +4,7 @@ const axios = require("axios");
 const bodyParser = require("body-parser");
 const mailer = require("./libraries/mailer");
 const OrderRepository = require("./data-access/order-repository");
-
+const PubsubHandler = require("./libraries/pubsub");
```

Comment:
I'm not sure whether the API should speak with the MQ - They are two different doors to our application. One may want to test the/with MQ even though it doesn't have an API. There are Microservices without API.

One more question to ask ourselves - Is this a component test that checks some business features (delete order which is commanded via MQ) or do we conduct narrow integration tests over the MQ wrapper (ensure that the wrapper works fine)? I suggest to at least show some examples of business features, not sure about the 2nd option

I included below two example tests of business features